pcsx2-qt/MainWindow.ui

```diff
@@ -274,7 +274,7 @@
      <normaloff>.</normaloff>.</iconset>
    </property>
    <property name="text">
-    <string comment="In Toolbar">Start &amp;File...</string>
+    <string comment="In Toolbar">Start File...</string>
```

Comment:
Probably should drop the ellipsis from these too (I guess the toolbar caption removes it automatically, but better to be specific)